tests/lib/state_test.py

```diff
@@ -1293,3 +1295,265 @@ def _create_desired_state():
                 ]
             }
         )
+
+
+class TestLinuxBridgeState:
```

Comment:
shouldn't this test be skipped / xfailed until the implementation is added ?

Reply:
I did `cherry-pick` on all the patches except for `linux bridge, vlan filtering: activate port vlan feature`. AFAICS, these tests are not testing any kind of NM implementation so they are not failing without the implementation.

Indeed, they were not marked as skipped/xfailed originally so.. I don't think so.

Reply:
touchè :)